Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -93,17 +103,30 @@
 		806F1E3726B85367007A60E6 /* Demo */ = {
 			isa = PBXGroup;
 			children = (
+				806F1E4126B85369007A60E6 /* Assets.xcassets */,
+				806F1E4626B85369007A60E6 /* Info.plist */,
+				806F1E4326B85369007A60E6 /* LaunchScreen.storyboard */,
 				806F1E3826B85367007A60E6 /* AppDelegate.swift */,
 				806F1E3A26B85367007A60E6 /* SceneDelegate.swift */,
 				806F1E3C26B85367007A60E6 /* ViewController.swift */,
-				806F1E3E26B85367007A60E6 /* Main.storyboard */,
-				806F1E4126B85369007A60E6 /* Assets.xcassets */,
-				806F1E4326B85369007A60E6 /* LaunchScreen.storyboard */,
-				806F1E4626B85369007A60E6 /* Info.plist */,
+				80F33CE726F8DE29006811B1 /* DemoMerchantAPI.swift */,
+				80F33CF226F8EA50006811B1 /* DemoSettings.swift */,
+				BE1766B226F911A2007EF438 /* URLResponseError.swift */,
```

Comment:
Maybe tuck the `DemoMerchantAPI` and `URLResponseError` in a networking directory, since we're gonna have a lot more UI types / complexity introduced as we flesh out the demo app